src/main/resources/function/getFunctionConfiguration.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ WSO2 Inc. licenses this file to you under the Apache License,
+  ~ Version 2.0 (the "License"); you may not use this file except
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied. See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+<template xmlns="http://ws.apache.org/ns/synapse" name="getFunctionConfiguration">
+    <parameter name="functionName" description="Name of the Lambda function."/>
+    <parameter name="qualifier" description="version/alias to specify"/>
+    <parameter name="apiVersionGetFunctionConfiguration"
+               description="API version used for GetFunctionConfiguration method."/>
+    <sequence>
+        <property name="uri.var.functionName" expression="$func:functionName"/>
+        <property name="uri.var.qualifier" expression="$func:qualifier"/>
+        <property name="uri.var.apiVersion" expression="$func:apiVersionGetFunctionConfiguration"/>
+        <property name="httpMethod" value="GET"/>
+        <property name="uri.var.uriRemainder"
+                  expression="fn:concat('/', $ctx:uri.var.apiVersion, '/functions/', $ctx:uri.var.functionName,
+                   '/configuration')"/>
+        <script language="js">
+            <![CDATA[
+                var queryParameter = '';
+                var qualifier = mc.getProperty('uri.var.qualifier');
+
+                if (qualifier != null && qualifier != "")
+                {
+                    queryParameter = "?Qualifier=" + encodeURIComponent(qualifier);
+                }
+                mc.setProperty('uri.var.queryParameter', queryParameter.toString());
+            ]]>
+        </script>
+        <class name="org.wso2.carbon.esb.connector.auth.AmazonLambdaAuthConnector"/>
+        <header name="Authorization" scope="transport" expression="$ctx:authorizationHeader"/>
+        <header name="X-Amz-Date" scope="transport" expression="$ctx:x-amz-date"/>
+        <payloadFactory media-type="json">
+            <format/>
+            <args/>
+        </payloadFactory>
```

Comment:
Do you need payloadFactory mediator here?

Reply:
Fix with d1edbe1

Reply:
Fix with 408c7bd